test/test_schmidt.py

```diff
@@ -75,6 +75,17 @@ def test_initialize_full_rank(self):
 
         self.assertTrue(np.allclose(state_vector, state))
 
+    def test_initialize_ame(self):
```

Comment:
```suggestion
    def test_initialize_ame(self):
        """ Test initialization of a maximum entangled state"""
```

Reply:
AME = Absolutely Maximally Entangled. I will include the comment accordingly.